README.md

```diff
@@ -34,13 +34,15 @@ First, create the steps container with a selector easily accessible.
 <div id="steps"></div>
 ```
 
-Next, we will create steps. For our example, two steps `People` and `Planet`. All steps needs inheritance from `Steps` to access hook functions.
+Next, we will create the steps `People` and `Planet` for our example. All the steps needs inheritance from `Steps` to access hook functions.
```

Comment:
Est-ce que tu souhaites faire l'exemple avec les gens ou émettre des directives pour expliquer un exemple ? 

Pas de `s` à `need` :
_All the steps need inheritance from..._

Reply:
Plutôt émettre des directives pour expliquer un exemple d'implémentation.
L'exemple que je décris dans le README est basé sur l'exemple SWAPI disponible dans le projet.
mais peut-être que c'est confusant ?

Reply:
Si émettre des directives, peut-être vaut-il mieux ne pas utiliser "we will...", mais directement, "Next, create the steps..."

Reply:
C'est corrigé